src/envoy/http/authn/http_filter_factory.cc

```diff
@@ -42,7 +42,7 @@ class AuthnFilterConfig : public NamedHttpFilterConfigFactory,
     if (status.ok()) {
       return createFilter(policy_);
     } else {
-      ENVOY_LOG(debug, "Utils::ParseJsonMessage() return value is NOT ok!");
+      PANIC("Utils::ParseJsonMessage() return value is NOT ok!");
```

Comment:
One more small request, can you also put status code and/or message to the panic text.

Reply:
Done.